modules/sdk-coin-ada/src/lib/transactionBuilder.ts

```diff
@@ -152,6 +231,15 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     this._transaction = transaction;
   }
 
+  /** @inheritdoc */
+  get getFee(): string {
+    return this._fee.to_str();
+  }
+
+  get getChangeAddress(): string {
```

Comment:
do we need this method?

Reply:
I just use it to check the fee in the unit tests, which I think is useful, if you think it's not, I will delete